examples/custom_key_bindings.py

```diff
@@ -0,0 +1,32 @@
+"""
+Display one 4-D image layer using the add_image API
+"""
+
+import numpy as np
+from skimage import data
+from napari import ViewerApp
+from napari.util import app_context
+
+
+with app_context():
+    viewer = ViewerApp()
+    blobs = data.binary_blobs(length=128, blob_size_fraction=0.05,
+                              n_dim=2, volume_fraction=.25).astype(float)
+
+    viewer.add_image(blobs)
+
+    def accept_image(viewer):
+        print('this is a good image')
+        next(viewer)
+
+    def reject_image(viewer):
+        print('this is a bad image')
+        next(viewer)
+
+    def next(viewer):
+        blobs = data.binary_blobs(length=128, blob_size_fraction=0.05,
```

Comment:
I'm not sure how much we are worried about demo performance, but this is pretty latent on my computer. I profiled it and it usually only takes ~10 ms on my comp (mostly getting the blob image), but it feels really slow for some reason. Also it sometimes pinwheels, but I can't get it to reproducibly do that so I'm not sure what that is.

Do you experience this on your computer?

Reply:
I don't get the pinwheel, but it does seem a little slow. ~10 ms should be ok though